src/helpers/email/send.ts

```diff
@@ -71,26 +71,28 @@ export async function newRequest(requestObj: any): Promise<any> {
 			if (!manager) {
 				return false;
 			} else if (manager.emails) {
-				let msg = requestTemplate(
-					requestObj.quantity,
-					requestingUser.name,
-					manager.name,
-					item.name,
-					store.name,
-					store._id,
-				);
-				let email = {
-					from: `ETHS Kitcoin Team <${email_from}>`,
-					replyTo: email_replyto,
-					to: manager.email,
-					subject:
-						`${requestingUser.name} requested ` +
-						(requestObj.quantity > 1 ? 'items!' : 'an item!'),
-					html: msg,
-				};
-				await transporter.sendMail(email).catch(console.log);
+				emails.push(manager.email!);
 			}
 		}
+		let msg: string = requestTemplate(
```

Comment:
Minor, but you don't need to specify that `msg` is a string because typescript already assumes that (based on the function signature)